app/src/main/res/layout/activity_main.xml

```diff
@@ -6,9 +6,32 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <androidx.fragment.app.FragmentContainerView
-        android:id="@+id/fragment_container_view"
+    <TextView
+        android:id="@+id/header"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="To-Do List"
```

Comment:
Since we're probably not setting up a localization API for this app, a better practice would be to use the `strings.xml` file to store the strings we use in views.